src/views/trade_history_view.js

```diff
@@ -0,0 +1,30 @@
+import Backbone from 'backbone';
+
+const TradeHistoryView = Backbone.View.extend({
+  initialize(params) {
+    this.bus = params.bus;
+    this.template = params.template,
+    this.listenTo(this.bus, 'bought', this.newBuy);
+    this.listenTo(this.bus, 'sold', this.newSell);
```

Comment:
This is a good use of the bus. There are many sources for the same type of event (quotes being immediately bought or sold, and orders executing when the price is right) and we want to treat them all the same, so we have them all trigger events on the same object. This greatly simplifies the design.